datahub-frontend/run/run-local-frontend-debug

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+CURRENT_DIR=$(pwd)
+BUILD_DIR=../build/stage/datahub-frontend
+CONF_DIR=$BUILD_DIR/conf
+
+set -a
+source frontend.env
+set +a
+
+export JAVA_OPTS="
+	-Xms512m
+	-Xmx1024m
+	-Dhttp.port=$PORT
+	-Dconfig.file=$CONF_DIR/application.conf
+	-Djava.security.auth.login.config=$CONF_DIR/jaas.conf
+	-Dlogback.configurationFile=$CURRENT_DIR/logback.xml
+	-Dlogback.debug=true
+	-agentlib:jdwp=transport=dt_socket,server=y,suspend=y,address=5005"
```

Comment:
intentionally committing this file to ease debug dev experience from now on.

Reply:
thank you!